Tribler/Core/Modules/restapi/multichain_endpoint.py

```diff
@@ -0,0 +1,77 @@
+import json
+
+from twisted.web import http, resource
+
+from Tribler.community.multichain.community import MultiChainCommunity
+
+
+class MultichainEndpoint(resource.Resource):
+    """
+    This endpoint is responsible for handing requests for multichain data.
+    """
+
+    def __init__(self, session):
+        resource.Resource.__init__(self)
+
+        child_handler_dict = {"stats": MultichainStatsEndpoint}
+
+        for path, child_cls in child_handler_dict.iteritems():
+            self.putChild(path, child_cls(session))
+
+
+class MultichainStatsEndpoint(resource.Resource):
```

Comment:
Hummm, interresting point. I'll admit I used your DebugEndpoint as a template for this resource. Mainly I thought it verly likely that there would be further resources in the future. For example /multichain/block/<id>. Or validation results. So I decided to make the /multichain/stats (or /statistics) a separately named resource as opposed to having just a /multichain resource that always serves the stats.

Then the placement under /debug. What is your reasoning about placing the multichain statistics under the /debug resource? I'm not convinced this resource is purely diagnostic. I already have a GUI usecase for it.